app/models/manageiq/providers/redhat/infra_manager/inventory/strategies/v4.rb

```diff
@@ -17,7 +17,7 @@ def host_targeted_refresh(target)
     end
 
     def vm_targeted_refresh(target)
-      ems.with_provider_connection(:version => 4) do |connection|
+      @ems.with_provider_connection(:version => 4) do |connection|
```

Comment:
@borod108 
Could we have used the old code as is because ems is an attr_reader method and it would have always returned @ems.

This also applies to the other @ems changes in this file.